Presentation.Web/Models/API/V1/Organizations/StsOrganizationChangeLogResponseDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Core.DomainModel.Organization;
+using Presentation.Web.Controllers.API.V1.Mapping;
+
+namespace Presentation.Web.Models.API.V1.Organizations
+{
+    public class StsOrganizationChangeLogResponseDTO
+    {
+        public StsOrganizationChangeLogOriginOption Origin { get; set; }
+        public string? Name { get; set; }
```

Comment:
why add nullable to the string?... it is a reference type and already nullable. We did not enable nullable compiler checks and that's what the "?" is for in C# - no runtime behavior changes

Reply:
Once migrated to a user you chan change it to a UserWithEmailDTO